### PR TITLE
Run `entities-service validate` as a pre-commit hook

### PR DESCRIPTION
**Pull request by CasperWA**
Add a pre-commit hook with id `validate-entities`.
It runs on all known file formats as well as with `--verbose`.

One can add extra options through the pre-commit `args`.
Finally, in its current version, one _must_ add ``.[cli]'` to the pre-commit `additional_dependencies`.

### CODE REVIEW

**Review by Treesarj — APPROVED** [review]

**Comment by CasperWA** [comment]
The two commits 7bd888a and 1ced968 were what made up this PR.
Due to an extra merge commit in this PR branch' HEAD, GitHub could not recognize that this PR has been merged into `main`. Instead, when rebasing these commits onto main, it was found locally that this would result in a `noop` (indeed ...), and after pushing that, the PR has been closed.

TL;DR: This PR was successfully merged (not closed), adding the commits 7bd888a and 1ced968 to `main`.
